X16 Reference - 11 -  Working with CMDR-DOS.md

```diff
@@ -0,0 +1,368 @@
+
+There are three basic interfaces for CMDR-DOS: the binary interface (SAVE,LOAD), the data file interface (OPEN, 
+PRINT#, INPUT#, GET#), and the command interface. We will give a brief summary of BASIC commands here, but please refer
+to the BASIC chapter for full syntax of each command. 
+
+If you are familiar with the SD2IEC or the CMD hard drive, navigating partitions and subdirectories is similar, with "CD", "MD", and "RD" commands
+to navigate directories.
+
+## Binary Load/Save
+
+The primary use of the binary interface is loading and saving program files and loading binary files into RAM. 
+
+Your binary commands are LOAD, SAVE, BLOAD, VLOAD, and BVLOAD. 
+
+This is a brief summary of the BASIC disk commands LOAD and SAVE. For full documentation, refer to Chapter 3: BASIC 
+Programming.
+
+### LOAD
+
+`LOAD <filename> [,device][,secondary_address][,start_address]`
+
+This reads a program file from disk. The first two bytes of the file are the memory location to which the file will be
+loaded, with the low byte first. BASIC programs will start with $01 $08, which translates to $0801, the start of BASIC
+memory. 
+The device number should be 8 for reading from the SD card. 
+
+secondary_address has multiple meanings: 
+
+* 0 or not present: load the data to address $0801, regardless of the address header. 
+
+* 1: load to the address specified 
+
+* 2: load into VERA RAM bank 0 (at the 16-bit address in the file)
+
+* 3: load into VERA RAM bank 1 (at the 16-bit address in the file)
+
+* 17 or $11: load into VERA RAM bank 16 or $0F 
+
+start_address is the location to read your data into. If you need to relocate your data to banked RAM, for example, you 
+will want to set the address to $A000 or higher. 
+
+Examples:
+
+`LOAD "ROBOTS.PRG",8,1` loads the program "ROBOTS.PRG" into memory at the address encoded in the file. 
+
+`LOAD "HELLO",8` loads a program to the start of BASIC at $0801.
+
+`LOAD "*",8,1` loads the first program on the current directory. See the section below on wildcards for more 
+information about using * and ? to access files of a certain type or with unprintable characters. 
+
+`LOAD "DATA.BIN",8,1,$A000` loads a file into banked RAM, starting at $A000. Don't forget to set the bank first: bank 
+0 is used by the operating system, so 
+
+### SAVE 
+
+`SAVE <filename>[,device]`
+
+Save a file from the computer to the SD card. SAVE always reads from the beginning of BASIC memory at $0801, up to the
+end of the BASIC program. Device is optional and defaults to 8 (the SD card, or an IEC disk drive, if one is plugged in.) 
+
+One word of caution: CMDR-DOS will not let you overwrite a file by default. To overwrite a file, you need to prefix
+the filename with @:, like this: 
+
+`SAVE "@:DEMO.PRG"`
+
+You may need to save arbitrary binary data from other locations. To do this, use the S command in the MONITOR (Chapter 
+6: Machine Language Monitor). 
```

Comment:
We can use markdown syntax to hyperlink to other chapters.  You can see examples of this in the table of contents file

Reply:
Perhaps we should emphasize the importance of running `DOS` after a `SAVE`.  `SAVE` will always succeed on the BASIC side but DOS could have errored.

Reply:
agreed. Expanded with reasons.